contracts/core/libs/utils/Bitmap256Bit.sol

```diff
@@ -0,0 +1,111 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.14;
+
+/*
+ * Each attribute is encoded on 4 bits in the 256-bit bitmap.
+ * The 4 bits encode the attribute's value (from 0 to 15) 
+ * If the value is 0, the attribute is disabled. 
+
+    attribute63         attribute2   attribute1   attribute0
+    bits                bits         bits         bits 
+   ┌────────────┐      ┌────────────┬────────────┬────────────┐
+   │ 1  1  1  1 │ .... │ 1  0  1  1 │ 0  0  0  0 │ 1  0  0  1 │
+   └────────────┘      └────────────┴────────────┴────────────┘
+      value 15            value 11     value 0      value 1
+    
+  * There is a total of 64 attributes.
+  * The attribute index must be between 0 and 63.
+  * The attribute value must be between 0 and 15.
+  * An enabled attribute will always have a value greater than 0.
+  * A disabled attribute will always have a value equal to 0.
+ **/
+
+library Bitmap256Bit {
+  uint256 constant MAX_INT = 2 ** 256 - 1;
+
+  error AttributeIndexOutOfBounds(uint8 attributeIndex);
```

Comment:
If we layerize the "Bitmap256Bit" library to be only oriented around bit manipulation, we should not have the word "attribute" here